README.md

```diff
@@ -70,3 +67,12 @@ This template allows users to connect their data to DataRobot's functionality se
    ```bash
    kedro run
    ```
+
+### Running Tests
+To ensure that your assets are set up properly, we have tests set up.
+1. `pip install pytest pytest-cov`
+
+2. Ensure you're at the root of the directory where the project is being run from...
+For this project --> you should be in `.../recipe-weather-forecastic/recipe-weather-forecastic`
```

Comment:
Why `...` in this case?

Reply:
Just meant to signify that there's some upper level directory